api/src/main/java/org/openmrs/ui/framework/UiUtils.java

```diff
@@ -397,7 +397,20 @@ public String formatDatePretty(Date date) {
 			return format(dateExt.getDateWithoutTime());
 		}
 	}
-	
+
+	/**
+	 * Formats a date without any timezone conversion
+	 *
+	 * @param date the date to format
+	 * @return date formatted
+	 */
+	public String formatDateWithoutTimezoneConversion(Date date) {
+		String dateFormatGP = Context.getAdministrationService()
+				.getGlobalProperty(UiFrameworkConstants.GP_FORMATTER_DATE_FORMAT, "dd.MMM.yyyy");
+		SimpleDateFormat dateFormat = new SimpleDateFormat(dateFormatGP, Context.getLocale());
```

Comment:
The others did not pass the locale second parameter. Is there any reason why this time you are passing it?

Reply:
The second parameter is only a fallback if we don't have that GP.